Marlin/src/gcode/calibrate/G34_M422.cpp

```diff
@@ -238,6 +244,12 @@ void GcodeSuite::G34() {
     // After this operation the z position needs correction
     set_axis_is_not_at_home(Z_AXIS);
 
+    #if ENABLED(BLTOUCH) && ENABLED(BLTOUCH_HS_MODE)
```

Comment:
Honestly probably fine to leave the stow in regardless in case of any abort or error just to be safe.

Reply:
There wasn't one there before at all